### PR TITLE
LR M1 support note

### PR DESCRIPTION
**Pull request by anvascon**
LR M1 support note

### CODE REVIEW

File: microsoft-365/security/defender-endpoint/live-response.md [hunk 1]
```diff
@@ -57,6 +57,9 @@ Before you can initiate a session on a device, make sure you fulfill the followi
     - [Version 1709 (RS 3)](/windows/whats-new/whats-new-windows-10-version-1709) with [KB4537816](https://support.microsoft.com/help/4537816/windows-10-update-kb4537816)
 
   - **macOS** - Only applicable for Public Preview, minimum required version: 101.43.84 
+
+  > [!NOTE]
+  > Currently, only Intel based chipsets are supported.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/live-response.md]:
```suggestion
  > Currently, only Intel-based chipsets are supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @anvascon, kindly commit the minor suggested change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by mjcaparas** [comment]
This update is already live.

**Pull request closed without merging**